cumulusci/tasks/salesforce/sourcetracking.py

```diff
@@ -0,0 +1,109 @@
+from collections import defaultdict
+import os
+
+import yaml
+
+from cumulusci.core.exceptions import TaskOptionsError
+from cumulusci.salesforce_api.metadata import ApiRetrieveUnpackaged
+from cumulusci.tasks.salesforce import BaseRetrieveMetadata
+from cumulusci.tasks.salesforce import BaseSalesforceApiTask
+from cumulusci.tasks.metadata.package import PackageXmlGenerator
+from cumulusci.tasks.metadata.package import __location__
+
+
+class ListChanges(BaseSalesforceApiTask):
+    def _run_task(self):
```

Comment:
consider putting the list of changes by membertype into ListChanges.return_values ?

Reply:
I did consider it, but I figured I'd rather wait for a use case then guess at what data structure is going to be most helpful there.

Reply:
at some point i'm going to just add return_values to every task we have, because if we don't start adding them we'll never iterate on them or use them :) but I hear your point, so I won't make you do it